apps-rendering/package.json

```diff
@@ -20,6 +20,7 @@
 		"watch:client": "webpack serve --config webpack.config.ts --config-name client",
 		"watch:cdk": "tsc -w -p config/tsconfig.cdk.json",
 		"watch": "yarn watch:server & yarn watch:client",
+		"watch:simulator": "PORT=3030 yarn watch",
```

Comment:
Just wondering, what's the reason for having a separate port for the simulator? Is there a downside to switching the main script from `8080` over to `3030` and letting people know we've made the change?

Reply:
There's no advantage to a separate port for the simulator. Switching to `3030` for the dev server by default seems like a good option to me!